test/utils/service.go

```diff
@@ -0,0 +1,208 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"context"
+	"fmt"
+	topoapi "github.com/onosproject/onos-api/go/onos/topo"
+	"github.com/onosproject/onos-lib-go/pkg/errors"
+	"github.com/onosproject/onos-lib-go/pkg/logging"
+	"github.com/onosproject/onos-lib-go/pkg/northbound"
+	"github.com/onosproject/onos-topo/pkg/store"
+	"google.golang.org/grpc"
+)
+
+var log = logging.GetLogger("northbound", "topo")
+
+// NewService returns a new topo Service
+func NewService(store store.Store) northbound.Service {
```

Comment:
This is not needed. Please delete it. service is implemented in the NB of topo already.